codespell_lib/data/dictionary.txt

```diff
@@ -34744,6 +34763,10 @@ unsucccessful->unsuccessful
 unsucccessfully->unsuccessfully
 unsucccessul->unsuccessful
 unsucccessully->unsuccessfully
+unsuccee->unsucceed
+unsucceedde->unsucceeded
+unsucceeed->unsucceeded
+unsuccees->unsuccess, unsucceeds,
```

Comment:
I suggest you drop all of them. While `unsuccessful, unsuccessfully, succeed` are valid words. 

But `unsucceed, unsucceeded, unsuccess, unsucceeds` are not valid words.
They are invalid in aspell, hunspell, ispell, aoo-mozilla-en-dict, chrome browser and reputable online dictionaries.

https://www.lexico.com/search?filter=en_dictionary&query=unsucceed
https://www.ldoceonline.com/spellcheck/english/?q=unsucceed
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unsucceed

https://dictionary.cambridge.org/spellcheck/english/?q=unsucceed
https://www.dictionary.com/misspelling?term=unsucceed
https://www.merriam-webster.com/dictionary/unsucceed

https://www.lexico.com/search?filter=en_dictionary&query=unsucceeded
https://www.ldoceonline.com/spellcheck/english/?q=unsucceeded
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unsucceeded

https://dictionary.cambridge.org/spellcheck/english/?q=unsucceeded
https://www.merriam-webster.com/dictionary/unsucceeded

Reply:
Modified, please see [a7eaeaf](https://github.com/codespell-project/codespell/pull/2445/commits/a7eaeafe32fe4a7f87d69d46623fbc527a4a488a)